tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
I assume the root ignore file should cover these settings.

Reply:
Yeah - I'll clean this up.